apps/echo/server/src/main/java/net/discdd/app/echo/AdapterSecurityConfig.java

```diff
@@ -0,0 +1,39 @@
+package net.discdd.app.echo;
+
+import net.discdd.security.AdapterSecurity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.INFO;
+import static java.util.logging.Level.SEVERE;
+
+@Configuration
+public class AdapterSecurityConfig {
+    private static final Logger logger = Logger.getLogger(AdapterSecurityConfig.class.getName());
+
+    @Autowired
+    ApplicationContext context;
+
+    @Value("${echo-server.rootdir}")
+    private Path adapterPath;
+
+    @Bean
+    AdapterSecurity createAdapterSecurityInstance() {
+        try {
+            logger.log(INFO, "Creating adapter security instance");
+
+            return AdapterSecurity.getInstance(adapterPath);
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
```

Comment:
remove the autogenerated comment. pass the exception to the logger rather than printing the stack track